examples/simulated/plot_riemannian_gaussian.py

```diff
@@ -0,0 +1,77 @@
+"""
+=====================================================================
```

Comment:
Currently, examples are not added in the gallery of pyRiemann: you have to add a readme file at the root of `examples/simulated`.

Reply:
Ok! I run the toy_classification on my laptop and it took 1 minute, so I check the average time of CI for building docs but it did not change much from the main branch. 
@plcrodrigues you may have to reduce the `n_samples` from make_gaussian_blocs or the `deltas_array` length if the computation is too long.

Reply:
You're right ! I downscaled things so to have an execution time of ~15 seconds. Sounds good to you?

Reply:
Yes!